internal/cli/clusters/update.go

```diff
@@ -59,6 +63,84 @@ func (opts *UpdateOpts) initStore(ctx context.Context) func() error {
 }
 
 func (opts *UpdateOpts) Run() error {
+	if opts.isFlexCluster {
+		return opts.RunFlexCluster()
+	}
+
+	return opts.RunDedicatedCluster()
+}
+
+func (opts *UpdateOpts) RunFlexCluster() error {
+	flexClusterReq, err := opts.newFlexCluster()
+	if err != nil {
+		return err
+	}
+
+	flexCluster, err = opts.store.UpdateFlexCluster(opts.ConfigProjectID(), opts.name, flexClusterReq)
+
+	apiError, ok := atlasv2.AsError(err)
+	code := apiError.GetErrorCode()
+	if ok {
+		if apiError.GetErrorCode() == invalidAttributeErrorCode && strings.Contains(apiError.GetDetail(), regionName) {
+			return cli.ErrNoRegionExistsTryCommand
+		}
+		if ok && code == duplicateClusterNameErrorCode {
```

Comment:
redundant `ok &&`, you're already in the `if ok {` branch

Reply:
updated